pkg/registry/service/ipallocator/allocator_test.go

```diff
@@ -142,27 +143,13 @@ func TestAllocateSmall(t *testing.T) {
 		}
 	}
 
-	if r.count != 2 && r.Free() != 0 && r.max != 2 {
+	if r.Free() != 0 {
```

Comment:
Added the test for max back in.  Still dropped the test for count, but given that Free = max - count, I hope that is OK!